src/components/PhonesCatalog/PhoneDetailsSingle/PhoneDetailsSingle.js

```diff
@@ -63,7 +66,8 @@ const Phone = (props) => {
 
 const mapStateToProps = (state) => ({
    images: state.imagesData,
```

Comment:
maybe
```suggestion
   images: state.imagesData.images,
```

and on line 11 to use like 
```js
   const { onImgFetch, phonesInfo, match, images } = props;

```